apps/api/src/vault/vault.service.ts

```diff
@@ -74,6 +72,18 @@ export class VaultService {
     }
   }
 
+  async checkVaultOwner(keycloakId: string, vaultId: string) {
+    const person = (await this.personService.findOneByKeycloakId(keycloakId)) as Person
```

Comment:
Same questions as above, what happens if the db cannot find that person, maybe we need to throw `UnauthorizedException` also

```suggestion
    const person = await this.personService.findOneByKeycloakId(keycloakId)
    if (!person) {
      throw new UnauthorizedException()
    }
```